src/tools/singlejar/combiners.cc

```diff
@@ -284,3 +295,152 @@ void *ManifestCombiner::OutputEntry(bool compress) {
   concatenator_->Append("\r\n");
   return concatenator_->OutputEntry(compress);
 }
+
+template<typename T>
```

Comment:
Would it be possible to move `Log4J2PluginDatCombiner` into a separate file?

Reply:
Extracted in 8e901f071a